tests/test_twodstats.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2016 by Mike Jarvis and the other collaborators on GitHub at
+# https://github.com/rmjarvis/Piff  All rights reserved.
+#
+# Piff is free software: Redistribution and use in source and binary forms
+# with or without modification, are permitted provided that the following
+# conditions are met:
+#
+# 1. Redistributions of source code must retain the above copyright notice, this
+#    list of conditions and the disclaimer given in the accompanying LICENSE
+#    file.
+# 2. Redistributions in binary form must reproduce the above copyright notice,
+#    this list of conditions and the disclaimer given in the documentation
+#    and/or other materials provided with the distribution.
+
+from __future__ import print_function
+import numpy as np
+import piff
+import os
+
+def test_twodstats():
+    """Make sure we can execute and print a readout of the plot
+    """
+
+    model = piff.Gaussian()
+    interp = piff.Polynomial(order=1)  # should find that order=1 is better
+    # create background model
+    stars, true_model = generate_starlist()
+    psf = piff.SimplePSF(model, interp)
+    psf.fit(stars, None, None)
+
+    stats = piff.TwoDHistStats(number_bins_u=21, number_bins_v=21, reducing_function='np.mean')
+    stats.compute(psf, stars)
+    # Test the plotting and writing
+    twodstats_file = os.path.join('output','twodstatsstats.png')
+    stats.write(twodstats_file)
+    # fig, axs = stats.plot()
+    import matplotlib.pyplot as plt
+    plt.show()
```

Comment:
This is not a valid unit test. 

1. There is no testing here. You need some actual asserts.  So far this just checks that there are no syntax errors in the code, which is a pretty low bar for a test.  
2. You can't have the test pop up a plot on the screen.  For one, people may not be running in an environment where that is even possible.  And moreover, we don't want to have to close this plot window every time we run nosetests.